wallet-connect/src/assets/logo.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
```

Comment:
Have we use assets/logo.png and logo.svg?

Reply:
Removed